[fix]: patch target/linux/generic/pending-6.1/705-net-dsa-tag_mtk-add-padding-for-tx-packets.patch

legale opened the pull request:
fix patch target/linux/generic/pending-6.1/705-net-dsa-tag_mtk-add-padding-for-tx-packets.patch

`git am` refuses to accept this patch. Change fixes it.

legale committed:
fix patch

Signed-off-by: Isaev Ruslan <legale.legale@gmail.com>

zxlhhyccc commented:
https://github.com/openwrt/openwrt/blob/main/target/linux/generic/backport-6.1/730-05-v6.3-net-dsa-tag_mtk-assign-per-port-queues.patch

hauke commented:
The patch in current master applies cleanly on top of the Linux kernel used in OpenWrt + the patches applied before. With your change it does not apply cleanly any more.

legale commented:
> The patch in current master applies cleanly on top of the Linux kernel used in OpenWrt + the patches applied before. With your change it does not apply cleanly any more. You should apply also all the patches before.

I'm trying to get external kernel tree for qualcommax target, so i'm applying with `git am` patches in this order:
- `target/linux/generic/backport-6.1`
- `target/linux/generic/pending-6.1`
- `target/linux/generic/hack-6.1`
- `target/linux/qualcommax/patches-6.1`

Will try to find what i have missed.

Thank you.

hauke commented:
The order looks correct. Are you using kernel 6.1.63 as a base?

legale commented:
> The order looks correct. Are you using kernel 6.1.63 as a base?
yep.
vanila v6.1.63 kernel

i've made `make target/linux/refresh`. Some patches was changed.

```
        modified:   target/linux/generic/pending-6.1/705-net-dsa-tag_mtk-add-padding-for-tx-packets.patch
        deleted:    target/linux/generic/pending-6.1/711-02-net-dsa-qca8k-enable-flooding-to-both-CPU-port.patch
        modified:   target/linux/generic/pending-6.1/711-03-net-dsa-qca8k-add-support-for-port_change_master.patch
        modified:   target/linux/generic/pending-6.1/712-net-dsa-qca8k-enable-assisted-learning-on-CPU-port.patch
        modified:   target/linux/qualcommax/patches-6.1/0101-clk-qcom-gcc-ipq8074-rework-nss_port5-6-clock-to-mul.patch
        modified:   target/linux/qualcommax/patches-6.1/0118-clk-qcom-Add-WCSSAON-reset.patch
```
